modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/decisions_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+    module V0
+        class DecisionsController < ApplicationController
```

Comment:
I'm especially curious where there is a specific customization that will give the route `.../decision`, but the controller name of `PowerOfAttorneyRequestDecisionsController`. This [guide](https://guides.rubyonrails.org/routing.html) should say so. This would be a good look.

Reply:
It was expecting decision controller name. Had to do this - resource :decision, only: :create, controller: 'power_of_attorney_request_decisions' to get it to accept it